maas-schemas/schemas/core/components/non-strict-example.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$id": "https://schemas.maas.global/core/components/non-strict-example.json",
+  "description": "Example of a schema which fails strict mode validation",
+  "type": "array",
+  "items": [{ "type": "string" }, { "type": "string" }],
```

Comment:
Is the keyword supposed to be `prefixItems`? In that case this would be a schema of a pair of strings. In its current form I would expect this to be an invalid schema. I don't think `items` can be an array of schemas.

Reply:
Also, perhaps add some examples.

Reply:
Removed